pyuvsim/telescope.py

```diff
@@ -3,12 +3,17 @@
 # Licensed under the 3-clause BSD License
 """Definition of Telescope objects, for metadata common to all antennas in an array."""
 
-import numpy as np
 import warnings
+
+import numpy as np
 from pyuvdata import UVBeam, parameter
 
+try:
+    from . import mpi
+except ImportError:
+    mpi = None
```

Comment:
Can this None cause issues for us further down in the file?

Reply:
I thought it was handled properly, but on further inspection I think there might be a problem. I'll push a fix.